packages/autorest.python/autorest/codegen/serializers/general_serializer.py

```diff
@@ -115,6 +115,11 @@ def serialize_setup_file(self) -> str:
         params = {}
         params.update(self.code_model.options)
         params.update(self.code_model.package_dependency)
+        params["extra_depends"] = []
```

Comment:
what is `extra_depends`?

Reply:
for adding "typing_extensions" in setup.py